### ISSUE TITLE
ssm_to_proteinpaint function needed

### ISSUE THREAD

**Issue by rdmorin** [morinlab/GAMBLR#249]
We need a function that takes a data frame in MAF format and generates an input file for [ProteinPaint](https://docs.google.com/document/d/1OJ9aXq2_-a3BfIQdKLYCYzrJRTpu4_9i3gephTY-Z38/edit) using the mutations in the MAF.

It should optionally add one or more of the optional columns that ProteinPaint can accept (using information in the GAMBL metadata).

**Comment by rdmorin** [comment]
@vladimirsouza has this been completed?

**Comment by vladimirsouza** [comment]
Yes. This has been addressed in [this](https://github.com/morinlab/GAMBLR.viz/pull/21) open PR and is waiting for approval.

**Comment by Kdreval** [comment]
Yes that PR is under review and there few outstanding things to address/improve. It was updated to resolve those and I will review it today :+1:

**Comment by HoumanLM** [comment]
I have two small suggestions: 
For the checking the columns and printing error message in case some of them are not in the file, maybe it would be better if we summarize the code to make it easier to understand. For example, we can change it to this code:
> if (any(!(maf_req_cols %in% names(maf_data)))) {
+     missing_cols <- maf_req_cols[!(maf_req_cols %in% names(maf_data))]
+     stop(paste("Required columns missing in the input MAF data frame:", paste(missing_cols, collapse = ", ")))
+ }

The other thing is about coding_only param which might be better if we make its default value to "TRUE" as we are going to consider variants in the context of proteins.

**Comment by mattssca** [comment]
Thanks, Houman! The PR is merged. Please post your suggestions for enhancement as a new issue under the correct repo ([GAMBLR.viz](https://github.com/morinlab/GAMBLR.viz)) for this function as a new issue and mark it with the **enhancement** tag (or any other tag you seem fit).

**Comment by HoumanLM** [comment]
Sure Adam! Thanks!

**Comment by Kdreval** [comment]
That PR has been merged
